### PR TITLE
Fixes #30001 - fix auth sources breadcrumbs

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: app/views/auth_source_ldaps/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,14 @@
-<% title _("Create LDAP Auth Source") %>
-
+<%= breadcrumbs(
+    items: [
+      {
+        caption: _("Authentication Sources"),
+        url: auth_sources_path
+      },
+      {
+        caption: 'Create LDAP Auth Source',
```
Comment (tbrisker) [inline app/views/auth_source_ldaps/new.html.erb]: This should be translated

File: app/views/auth_source_externals/edit.html.erb [hunk 2]
```diff
@@ -1,3 +1,14 @@
-<% title _("Edit External Authentication Source") %>
-
+<%= breadcrumbs(
+    items: [
+      {
+        caption: _("Authentication Sources"),
+        url: auth_sources_path
+      },
+      {
+        caption: _("Edit %s" % @auth_source_external.to_label),
```
Comment (tbrisker) [inline app/views/auth_source_externals/edit.html.erb]: There can only be one external source, the caption should remain the same

File: app/views/auth_source_ldaps/edit.html.erb [hunk 3]
```diff
@@ -1,3 +1,14 @@
-<% title _("Edit %s") % @auth_source_ldap.to_s %>
-
+<%= breadcrumbs(
+    items: [
+      {
+        caption: _("Authentication Sources"),
+        url: auth_sources_path
+      },
+      {
+        caption: _("Edit %s" % @auth_source_ldap.to_label),
```
Comment (tbrisker) [inline app/views/auth_source_ldaps/edit.html.erb]:
```suggestion
        caption: _("Edit %s") % @auth_source_ldap.to_label,
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
Thanks, a couple of small comments inline regarding the titles

**Comment by rabajaj0509** [comment]
Agree with @tbrisker! Thanks for fixing this. @amirfefer

File: app/views/auth_source_externals/edit.html.erb (hunk 2 above)
Comment (amirfefer) [reply app/views/auth_source_externals/edit.html.erb]: but you can rename it as you want, no?

Comment (tbrisker) [reply app/views/auth_source_externals/edit.html.erb]: not easily right now

**Review by tbrisker — APPROVED** [review]
Thanks @amirfefer !

**Comment by tbrisker** [comment]
2.1 - 64f54b442

**Pull request merged**